### DISCUSSION TITLE
Update the Rideframe Ui maybe for some tracks with new Curve size

### DISCUSSION THREAD

**Discussion by Realsteel89** [OpenRCT2/OpenRCT2#21107] [Feature requests] [answered]
On my maurer track aka "SpinningRollerCoaster" i like to have all 4 sizes of curves useable but the standard UI has only 3 radius buttons to use.
![grafik](https://github.com/OpenRCT2/OpenRCT2/assets/77458610/63030aa7-e89d-4ddd-abfb-49a2aa37fa3d)
so we had buttons for "1x1/2x2/3x3/4x4 tile cures" that every curve has 1 button not an double button soloution

Maybe later ideas:
- selection window for the type of lift, i.e. friction wheel or chain
- selection window for booster selection, i.e. friction wheel or LSM
- selection window for the Bremen/block brakes, i.e. brake shoes or magnetic brakes

**Comment by pfroud** [comment, marked as answer]
I'm linking pull request https://github.com/OpenRCT2/OpenRCT2/pull/21109 where this is being worked on :)
